client-api/src/main/java/org/ostelco/topup/api/core/Profile.java

```diff
@@ -5,9 +5,15 @@
 
 @Data
 public class Profile {
```

Comment:
Class should always have a short, concise class comment that describes the intent of the class